### PR TITLE
Collect screenshots for datasets

### PR DESCRIPTION
**Pull request by mtojek**
_I think it's better to review first: https://github.com/elastic/package-registry/pull/239_

This PR adds supports for collecting screenshots mentioned in 

Sample manifest.yml
```yaml
format_version: 1.0.0
name: activemq
title: Activemq
version: 0.0.1
license: basic
description: Activemq Integration
type: integration
categories:
- logs
- metrics
requirement:
  kibana:
    versions: ""
screenshots:
- src: img/filebeat-activemq-application-events.png
  title: Filebeat Activemq Application Events
  size: 4102x2424
  type: image/png
- src: img/filebeat-activemq-audit-events.png
  title: Filebeat Activemq Audit Events
  size: 4102x2480
  type: image/png
- src: img/metricbeat-activemq-broker-overview.png
  title: Metricbeat Activemq Broker Overview
  size: 2560x2032
  type: image/png
- src: img/metricbeat-activemq-queues-overview.png
  title: Metricbeat Activemq Queues Overview
  size: 2560x3712
  type: image/png
- src: img/metricbeat-activemq-topics-overview.png
  title: Metricbeat Activemq Topics Overview
  size: 2560x3712
  type: image/png
```

### CODE REVIEW

**Comment by ruflin** [comment]
@hbharding Lots of images for our future carousel ;-)

File: dev/import-beats/main.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ func main() {
 	// Target public directory where the generated packages should end up in
 	var outputDir string
 
-	flag.StringVar(&beatsDir, "beatsDir", "", "Path to the beats repository")
-	flag.StringVar(&outputDir, "outputDir", "", "Path to the output directory ")
+	flag.StringVar(&beatsDir, "beatsDir", "../beats", "Path to the beats repository")
```
Comment (ruflin) [inline dev/import-beats/main.go]: Nice default, makes it easy to run.

File: dev/import-beats/images.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+)
+
+type imageContent struct {
+	source string
+}
+
+func createImages(beatDocsPath, modulePath string) ([]imageContent, error) {
+	var images []imageContent
+
+	moduleDocsPath := path.Join(modulePath, "_meta", "docs.asciidoc")
```
Comment (ruflin) [inline dev/import-beats/images.go]: Nice idea to grab it from the docs.

**Comment by hbharding** [comment]
Great! We'll probably want to update a bunch of them when we get closer to GA, and when we decide to actually build the image carousel component. I think I've seen screenshots in the current "Add Data" UX that still show Kibana 6.x images.

**Comment by ruflin** [comment]
@hbharding Yes. For now I'm happy we have the screenshots :-) Also we should have a guideline then on how to create the screenshots.

**Pull request merged**